plugins/woocommerce/src/Admin/PluginsHelper.php

```diff
@@ -821,6 +821,35 @@ function ( $sub ) {
 
 		$total_expiring_subscriptions = count( $expiring_subscriptions );
 
+		// When payment method is missing on WooCommerce.com.
+		$helper_notices = WC_Helper::get_notices();
```

Comment:
this notice will be keyed (for dismissal / reshow purpose) with the same key as the expiring subscription notices key. Also this notice will use the same Track events as expiring subscription notices. Is this intentional?

Reply:
Yes, it's intentional as we'd like to show at most 1 notice and payment method only relevant when the subscription is expiring. The priority of what notice gets rendered will be:

1. Expired
2. Expiring and missing payment method
3. Expiring